test/Runtime/allocator_sanity.swift

```diff
@@ -18,10 +18,18 @@ AllocationsTestSuite.test("absurd.allocation.misaligned") {
 
 AllocationsTestSuite.test("absurd.allocation.gargantuan") {
   expectCrashLater()
+  // There is a chance that we'll actually be able to allocate Int.max bytes on
+  // 32-bit platforms, since they often have 4GB address spaces and Int.max is
+  // 2GB minus one byte. Allocate twice to ensure failure. That will (attempt
+  // to) allocate 4GB minus two bytes, and we'll definitely have more than two
```

Comment:
Unless the platform uses separate address spaces for code and data, in which case…

Reply:
If we find a situation where the system has less than three bytes allocated when this test runs, we can revisit it then. 😄